### DISCUSSION TITLE
Scheduler last heartbeat received last xx minutes ago

### DISCUSSION THREAD

**Discussion by krishnakanth1666** [apache/airflow#32083] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

Airflow Version - 2.0.1
Scheduler was going abruptly down with the below errors.

Waiting up to 5 seconds for process to exit...
Sending Signals.SIGTERM to GPID XXX
Sending Signals.SIGTERM to GPID XXX
Exited execute loop
Failed to execute task LocalTaskJob received SIGTERM signal.

After this message,the scheduler goes down everytime

### What you think should happen instead

I request you to provide some guidance to this issue

### How to reproduce

Not sure

### Operating System

cent os 

### Deployment

Other

### Deployment details

hosted airflow on google cloud instance vm

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by hussein-awala** [comment]
Could you please consider upgrading your Airflow version to the latest one (2.6.2)? The current version you're using was released two years ago (on 2021-02-08), and it's highly probable that the bug or problem you're experiencing has been resolved in more recent versions.
